backend/src/main/java/com/example/backend/model/Workout.java

```diff
@@ -1,14 +1,14 @@
 package com.example.backend.model;
+
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
 @Document("workout")
-public record Workout (
+public record Workout(
 
         @Id
-        String id ,
-
-        String description ,
+        String id,
+        String description,
         String title
 
 ) {
```

Comment:
Please use a DTO for Data coming from the frontend. See SonarCloud vulnerability-warning